<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KstChooseColorDialog</class>
 <widget class="QDialog" name="KstChooseColorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Assign Curve Color Per File</string>
  </property>
  <layout class="QGridLayout">
   <item row="1" column="0">
    <widget class="QGroupBox" name="vectorGroup">
     <property name="title">
      <string>Base Curve Color On</string>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="0">
       <widget class="QRadioButton" name="xVector">
        <property name="text">
         <string>X Vector</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="yVector">
        <property name="text">
         <string>Y Vector</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <layout class="QGridLayout">
     <item row="0" column="2">
      <widget class="QPushButton" name="Cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="OK">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <spacer name="spacer1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>160</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="_applyToNewCurves">
     <property name="text">
      <string>Apply to new curves</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="colorGroup">
     <property name="title">
      <string>Assign Colors</string>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="0">
       <widget class="QFrame" name="colorFrame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <tabstops>
  <tabstop>xVector</tabstop>
  <tabstop>yVector</tabstop>
  <tabstop>_applyToNewCurves</tabstop>
  <tabstop>OK</tabstop>
  <tabstop>Cancel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>Cancel</sender>
   <signal>clicked()</signal>
   <receiver>KstChooseColorDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
